Storage/rest/basic.h

```diff
@@ -123,7 +123,8 @@ struct Basic {
       input.entry.InitializeOwnKey();
       if (!Exists(input.field[current::storage::sfinae::GetKey(input.entry)])) {
         input.field.Add(input.entry);
-        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)), HTTPResponseCode.Created);
+        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)),
```

Comment:
I'd expect
`{ success: true, message: "Created.\n", data: { id: "new-id" } }`
or
`{ success: true, message: "Created.\n", id: "new-id" }`
but not
`{ success: true, message: "new-id" }`.